blog/2023-10-20-rebranding.mdx

```diff
@@ -0,0 +1,57 @@
+---
+slug: tolgee-rebranding
+title: "Tolgee's Exciting Transformation: A Modern, Fresh Look"
+description: Tolgee rebranding to new colors. We choose pink for it's modern and vibrant tone.
+authors: [sgranat]
+tags: ['brand', 'pink', 'rebranding', 'tolgee']
+---
+
+import { ReviewAuthor } from '../platform/shared/_ReviewAuthor';
+import { ScreenshotWrapper } from '../platform/shared/_ScreenshotWrapper';
+
+We're thrilled to introduce our refreshed brand, including our logo in a vibrant shade of pink, and a modern, user-friendly design for our app and website. We've heard your feedback, and we've taken it to heart.
+
+![New Tolgee landing page](/img/blog/rebranding/tolgee-landing-page.png)
+
+
+<!-- truncate-->
+
+## A New Look for a Bright Future
+
+Our journey to reimagining Tolgee began with a critical examination of our interface, and the consensus was clear: our old toolbar and color scheme just didn't cut it. So, we took the opportunity to overhaul the entire application's design, creating a cohesive and contemporary user experience.
+
+<ReviewAuthor
+  author={{
+    imageURL: '/img/blog/profile-photos/placeholder.png',
+    name: 'Martin P.',
+    title: 'Freelancer',
+  }}
+  description="The application doesn't have any bigger issues, just the design of UI is a little old-school."
+  capterraReview="https://www.capterra.com/p/10002120/Tolgee/reviews/4342787/"
+/>  
+
+Your feedback is invaluable, and we want to thank our users for their insights. The transformation of Tolgee is a direct response to your needs, and we hope it reflects our commitment to making your localization projects smoother and more enjoyable.
```

Comment:
I'd rewrite it as following: We want to thank to all our users for sharing their thoughts with us. The changes in Tolgee were made with you in mind as a direct response to your suggestions. We hope these updates will make your localization projects easier and more enjoyable.